images/oc-build-deploy-dind/build-deploy-docker-compose.sh

```diff
@@ -612,6 +612,7 @@ do
         oc get --insecure-skip-tls-verify -n ${OPENSHIFT_PROJECT} secret ${SERVICE_NAME}-servicebroker-credentials -o yaml > /oc-build-deploy/lagoon/${SERVICE_NAME}-servicebroker-credentials.yml
         set +x
         DB_HOST=$(cat /oc-build-deploy/lagoon/${SERVICE_NAME}-servicebroker-credentials.yml | shyaml get-value data.DB_HOST | base64 -d)
+        DB_READREPLICA_HOST=$(cat /oc-build-deploy/lagoon/${SERVICE_NAME}-servicebroker-credentials.yml | shyaml get-value data.DB_READREPLICA_HOST | base64 -d)
```

Comment:
@smlx 
What happens if there is no `data.DB_READREPLICA_HOST` in that yaml? I think we need to test for this first and only add the configmap entry if that actually was existign?

Reply:
good point, this will fail the deployment due to `pipefail`. I'll rework it.